denoise.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python

```

Comment:
We probably won't need this since we will be calling this with the python interpreter
### e.g. example usage
```bash
python api.py
```